### ISSUE TITLE
Highscore list shows more than 10 entries

### ISSUE THREAD

**Issue by Pohli** [Tomin1/trites#2]
If the highscore list has ten entries and one achieves a lower score than the 10th place, that new low score is displayed. It seems that all the low scores are stored, too.

![Screenshot_20241013_001](https://github.com/user-attachments/assets/08e6b38b-bf75-4a75-ba9f-5709a59aa8b3)

Xperia 10 III SFOS 4.5.0.24, Trites v1.0.1

**Comment by Tomin1** [comment]
Yeah, this is by design. I chose to show how the result compares to all previous results. I don't think anyone can fill their database so bad by playing the game that it would slow things down very much.

Of course it would be also an option to limit the number of results to something reasonable (100, 1000, 10 000, 100 000, a million, or something like that) and then if it's worse than that, just say that it's (e.g.) 1 000 000+ and not save it instead of storing everything. It's only a matter of someone implementing that feature. :slightly_smiling_face:
